src/app/store/rubrics.reducer.ts

```diff
@@ -17,7 +17,7 @@ export function rubricsReducer(state: RubricsStateInterface, action: RubricsActi
         case RubricsActionTypes.GetRubricSuccess:
         case RubricsActionTypes.UpdateCurrentRubric:
         case RubricsActionTypes.CreateRubricSuccess:
-            return {
+        return {
```

Comment:
Not sure how this indentation got changed, but do you mind changing it back @hollielu?

I'm also not sure why our linter didn't see it, but it's a good idea to keep indents consistent within files.